pyproject.toml

```
@@ -150,6 +150,18 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
+current_version = "2023.0.0"
+version_pattern = "YYYY.MM.INC0[-TAG]"
+commit_message = """
+Bump version to {new_version}
```

```suggestion
Bump version to {new_version}
"""

```
And the multi-line string needs to have an ending as well